masterlist.yaml

```
@@ -11321,6 +11321,17 @@ plugins:
         util: '[TES4Edit v4.0.4b](https://www.nexusmods.com/oblivion/mods/11536)'
         itm: 10
 
+  - name: 'Edible Slaughterfish.esp'
+    url:
+      - link: 'https://www.nexusmods.com/oblivion/mods/51975/'
+        name: 'Edible Slaughterfish'
+    tag:
+      - Invent.Add
+      - Scripts
+    clean:
+      - crc: 0xF3858C4A
+        util: 'TES4Edit v4.0.4b'
```

The author has updated the mod since this PR was submitted; this is the cleaning data for v1.2:

```suggestion
    clean:
      - crc: 0x7EF4DACB
        util: 'TES4Edit v4.0.4b'
```